code/__HELPERS/files.dm

```diff
@@ -72,17 +72,11 @@
 /proc/pathflatten(path)
 	return replacetext(path, "/", "_")
 
-/// Returns the md5 of a file at a given path.
-/proc/md5filepath(path)
-	. = md5(file(path))
-
-/// Save file as an external file then md5 it.
+/// Grabs md5 checksum from a file and rehashes it with config specified salt ( md5(md5(file) + salt) )
 /// Used because md5ing files stored in the rsc sometimes gives incorrect md5 results.
```

Comment:
What about the comments? Seems like the whole "save to external file" bit was important.

The notch is less important since I doubt we'll ever use rust_g's MD5 hashing, but I'm unsure about the RSC thing.

Reply:
comment wasn't really important because it instantly deletes the file after grabbing the hash

Reply:
> /// Used because md5ing files stored in the rsc sometimes gives incorrect md5 results.